source/client/UICardWrapper.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, this_card, loc_xy):
         self.card = this_card
         self.img = UICardWrapper.getImage(self.card)
         self.img_clickable = ClickImg(self.img, loc_xy[0], loc_xy[1], self.img.get_width(), self.img.get_height(), 0)
-        self.selected = False
+        self.status = 0     # 0 = not selected or prepared, 1 = selected, 2 = prepared (ready for play)
```

Comment:
I highly recommend putting that comment on its own line so its more obvious, maybe even make blocks around it. It also should probably end up in docs about our UI eventually (make a task for later?)